source/utilities.cc

```diff
@@ -1501,9 +1501,10 @@ namespace aspect
     void
     AsciiDataBase<dim>::declare_parameters (ParameterHandler  &prm,
                                             const std::string &default_directory,
-                                            const std::string &default_filename)
+                                            const std::string &default_filename,
+                                            const std::string &default_subsection_name)
     {
-      prm.enter_subsection ("Ascii data model");
+      prm.enter_subsection (default_subsection_name);
```

Comment:
I think you chose the variable name incorrectly. It is not the "default name", but this really is the *name* of the section. So maybe the name of the function argument should be "subsection_name", and it can of course have a default value, but the name of the variable should no longer contain the part "default".